oak/server/rust/oak_runtime/build.rs

```diff
@@ -19,6 +19,7 @@ use std::path;
 fn main() {
     let proto_dir = path::Path::new("../../../../oak/proto/");
     let application_path = &*proto_dir.join("application.proto");
+    let logger_path = &*proto_dir.join("log.proto");
 
     // Tell Cargo that if the given file changes, to rerun this build script.
     // https://doc.rust-lang.org/cargo/reference/build-scripts.html#rerun-if-changed
```

Comment:
Should I add a `rerun-if-changed` for `log.proto` too?

Reply:
Yes, I think so.  I've got a change in #715 that should make keeping things in sync easier (but it does mean that whichever of the 2 PRs goes in second will need a minor manual merge).

Reply:
Thanks. I'll have a look at #715. This is a tiny PR so the manual merge should not be too difficult.